### DISCUSSION TITLE
[Scam Alert] Treenvest تری ان وست

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#29] [Announcements]
<div dir="rtl">

**Project Name:** 
Treenvest

**نام پروژه:** 
تری ان وست

**نماد پروژه:**

**اتفاقی که افتاده است:**
شرکت treenvest
نیز در رسته شرکت های پانزی قرار میگیرد و همچنین در بعضی از پلن های درامدزایی در رسته شرکت های هرمی قرار میگیرد
با گستردگی فعالیت در کامیونیتی های فارسی زبان میتوان این شرکت را در لیست هشدار قرار داد
ماهیانه سود ثابت می دهند.

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
طرح پانزی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
https://www.treenvest.com/
https://t.me/Treenvestir
https://t.me/Treenvestiran
https://www.instagram.com/iran.treenvest/?hl=en
https://instagram.com/iran.treenvest?utm_medium=copy_link
https://instagram.com/treenvestir?utm_medium=copy_link

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by saeedkousha** [comment]
https://www.instagram.com/iran.treenvest/?hl=en
 با سلام
طبق تحقیقات بنده شرکت  
treenvest 
نیز در رسته شرکت های پانزی قرار میگیرد و همچنین در بعضی از پلن های درامدزایی در رسته شرکت های هرمی قرار میگیرد
با گستردگی فعالیت در کامیونیتی های فارسی زبان میتوان این شرکت را در لیست هشدار قرار داد

**Comment by saeedkousha** [comment]
https://instagram.com/iran.treenvest?utm_medium=copy_link
https://instagram.com/treenvestir?utm_medium=copy_link

**Reply by davoodkiani** [reply]
اتفاقا خیلی هم خوبه . 10درصد سود میده ظرف 10ماه شما پول خودتو گرفتی .. شما که دلت میسوزه واسه اونای که فرمودید مردم پولشون با زحمت بدست میارن اینجوری از دست می‌دهند وود...میشه بفرماید کجا سپرده کنند ...تو بانکهای ایران ؟با دو درصد سود؟؟ تازه اونجا  باید نگران باشی که بانک کلاه تون رو بر نذاره ...البته با سودی که بر حسب ریال میپردازن و درصد خنده دار کلاهتون رو بر نمی‌دارند خیالتون راحت

**Reply by Blockunity** [reply]
امثال شما که از خون خواهد برادر و خانواده و فامیلتون تغذیه میکنید و پولی که زحمت کشیدن رو مثل زالو میکشید تو جیب خودتون، میشید مثل همون افرادی که با یونیک فایننس کل زندگی اطرافیان اشون رو اتیش زدند و زمانی که میگفتیم اسکمه مثل تو و امثال تو جواب میدادند.
شما ها باید مالباخته بشید تا بفهمید چکار میکنید.
برو سود پول خون اطرافیان ات رو برداشت کن و بگو پول دراوردم و اخرش خرج دوا و درمون میشه.
وقتی جلو چشمت یکی از اطرافیان ات برای فشار مالی خود کشی کرد میفهمی.
کثافت شدید بیزاریم از شما ادم های زالو و انسان نما

**Reply by Mohammeddsh1** [reply]
سلام من به این دلیلی که شما  تخصص دارید ازتون سوال  میپرسم  من توی این شرکت هستم و هنوز هم دارم در مورد این شرکت تحقیق میکنم ولی وقتی به پروژه هایی که ران کرده مثل بازی و ربات تریدر یه نشونه خوب و مثبته
و هیستوری ترید زنده رو هم برای دوست من فعال کرده و من دیدم میشه لطفا توضیح بدید در مورد اینها؟ اگر لازم باشه و بخواهید من میتونم یرای شما عکس  و مدرک ارسال کنم

**Reply by pooriasha** [reply]
سلام من خودم هم در این شرکت سه ماهه سرمایه گذاری کردم و قبل از اون یک ماه کامل وقت گذاشتم تحقیق کردم این شرکت چهل و سه صفحه وایت پیپر داره و طبق تصاویر ثبت شده در ربات های گوگل این سایت از سال ۲۰۲۰راه اندازی شده روش درامد زایی بصورت ترید هست که کاربران بصورت زنده میتوانند مشاهده کنند علاوه بر ربات های تریدر در این شرکت تریدر های باسابقه و ماهر هم فعالیت دارند از این نظر درامد شرکت مشکلی نداره و امروزه داره شرکت هایی ایرانی مثل ترینوست بوجود میاد مثل شرکت سکه بیت که دارای نماد اعتماد الکترونیک هست و ماهیانه هفت درصد به کاربرانش سود میده پس از این نظر درامد سایت هیچ مشکلی ندارد

**Reply by tnztzbfm** [reply]
> اتفاقا خیلی هم خوبه . 10درصد سود میده ظرف 10ماه شما پول خودتو گرفتی .. شما که دلت میسوزه واسه اونای که فرمودید مردم پولشون با زحمت میارن اینجوری از دست میدهند وود...میشه بفرماید کجا میگیرن ...تو بانکهای ایران ؟با دو درصد سود؟ ؟ تازه اونجا باید نگران باشی که بانک کلاه تون رو بر نذاره ...البته با سودی که بر حسب ریال میپردازن و درصد خنده دار کلاهتون رو بر نمیدارند خیالتون راحت.

فقط ایران دوزد هست
من سایت راضی هستم پرداختش عالی هست 💪👍

**Comment by Hosseinbhffjgmkgsjkjvsdhnv** [comment]
من مشغول فعالیت هستم.خیلی عالی هست و مورد خاصی ندیدم

**Comment by Hosseinbhffjgmkgsjkjvsdhnv** [comment]
ماهی 10% هر ماه داره پرداخت میکند
01.01.2020
راه اندازی شده

**Comment by micro73** [comment]
*بررسی قبل از جلسه پرزنت 
1. سایت پروژه به ادرس treenvest.com  :
       در مورد نحوه فعالیت توضیحی ازائه نشده – مدیران سایت و پروژه به هیچ عنوان مشخص نیستند- دفترمرکزی و یا ادرسی در سایت موجود نیست – پلن درامدی در سایت موجود نیست – قسمت ثبت نام ندارد و فقط ورود دارد.  
2. توکن 3co مربوط به پروژه 
          ادرس کانترکت توکن : 0x1DE64424B63ebAEc3a29108cb9965796ef4deB3c
         سایت توکن به ادرس 3co.net  
      در کوین مارکت کپ هنوز لیست نشده، پروزه به تازگی ران شده، قیمت و مارکت کپ ندارد- در دکس هنوز تامین نقدینگی هم نشده – صرفا فرد  سازنده توکن های تولید شده را در ولتی دارد و به قیمت فرضی خودش 1 سنت دارد  و شما به عنوان سود حاصل از سرمایه گزاری treenvest میتوانید این ارز را برداشت کنید.... در حال حاضر حدود 49 درصد توکن ها در دو ولت هست.   سوشیال مدیا نداردو فرد سازنده مشخص نیست. غیرقابل فروش هست.
** توضیحات ارائه شده توسط پرزنتور :::
برای ثبت نام برای هر فرد لینکی میفرستند که شامل کد دعوت نفربالایی است و میتواند ثبت نام کند.
طبق گفته پرزنتور با پول هایی که همه وارد میکنند دارند ترید میکنند و از اوون ترید سود به بقیه میدهند و لایو ترید برگزار میکنند که هویت هیچ کدام افراد مشخص نیست . 
انتخاب پلن 6 ماهه، 5 درصد ماهانه پاداش میدهد ولی همه پاداش ها اخر 6 ماه قابل برداشت است.
انتخاب پلن 1 ساله ، 7 در صد ماهانه پاداش میدهد و ماهانه قابل برداشت است.
انتخاب پلن 2 ساله، 10 درصد ماهانه پاداش میدهد و ماهانه قابل برداشت است.
در پلن 1 ساله و 2 ساله میتوان  اصل پول را از 2 ماه تا 9 ماه از ثبت جایگاه که گذشت برداشت کرد.
با ثبت هر شخص سه دست میتواند بسازد که هر دست میتواند شامل بی نهایت فرد باشد.      

90 روز از ثبت جایگاه هر فرد فرصت دارد با زدن 5000 هزاردلار در هردست 1000 دلار پاداش دریافت کند.
6 ماه از ثبت جایگاه هر فرد فرصت دارد تا سه تا 100000 در هر دست بزند و 30000 هزار دلار پاداش دریافت کند با زدن این قسمت سطح 4 تا 20 برایش فعال میشود و از هر سطح 0.5 0.4 0.3 تا لول 20 پاداش میدهد 
در ضمن 70 درصد سود را میتوانند ارز 3co  دریافت کنند.

سطح 1 که از هر دست 1% پاداش میدهد
سطح 2 که از هر دست 0.5% پاداش میدهد
سطح 3 که از هر دست 5% پاداش میدهد

یکی از موارد قابل توجه این است که قیمت توکن را به صورت فرضی 1 سنت در نظر گرفته‌اند و به اعضا می‌دهند

یکی از عجیب ترین کارهایی که بعضی از اعضا در حال انجامش هستند این است که در قبال سرمایه‌گذاری افراد جدید بهشون سند ملک، ماشین یا چک می‌دهند.

**Reply by mkordian** [reply]
من ۱۰۰۰ دلار سرمایه گذاری کردم، خیلی هم عالی هست، ۱۰ سال هست که دارن فعالیت میکنند، به نظر من که اصلا پانزی نیست و سود معقولی میده.

**Reply by Blockunity** [reply]
تو فوق العاده ای 🤣🤣 ۱۰ سال
میشه سابقه ۱۰ ساله اش رو بیاری؟ کریپتو کلا ۱۳ ساله تو مارکته🤣🤣🤣 ، حداقل ماله میکشید درست بکشید.

**Reply by alireza539** [reply]
هه 
تزه اينو نگفتم، منم يه قناري دارم رو پايي مي زنه و نون ميخره 🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣🤣

**Comment by micro73** [comment]
در جدیدترین صحبت با فردی که پرنتور هست و شبکه بالایی دارد اینگونه توجیه میکنند ::
چون این شرکت کار خیر میکنند و 10 در ضد سود میدهند خودشون رو پنهان میکنند چون هرکس کار خیر یکند بساطش رو جمع میکنند ..

در جواب ایشون گفتم مردم میخواهند سرمایه خودشون رو که با 1000 زحمت به دست میارن رو در اختیار جایی بزارن که هویتشون نامعلومه...

به هر شیوه ای هست مردم رو قانع میکنند که سرمایه بزارن و متاسفانه مردم بدون توجه و بررسی پول در چنین شرکت هایی میگذارند .

**Reply by davoodkiani** [reply]
( در جواب ایشون گفتم مردم میخواهند سرمایه خودشون رو که با 1000 زحمت به دست میارن رو در اختیار جایی بزارن که هویتشون نامعلومه...) بنده در جواب شما میگم  ک
ممنون بابت دلواپسی شما اگه امکان داره یه لطفی  کنید بگید مردم  کجا سرمایه گذاری کنند ...

**Reply by micro73** [reply]
سرمايهگذاري نياز به آموزش و تجربه كافي دارد 
شرايط و قوائد سرمايه گذاري رو بايد اموزش ديد 
من و شما قرارنيست براي كسي مشخص كنيم كجا بايد سرمايه گذاري كرد! 
هر شخص براي سرمايه گذاري بايد حداقل قوائد رو رعايت كند 
سوالي دارم،هميشه پول دست اشخاصي ميدي كه نميشناسي و نديدي؟ 
يا پلتفرم هايي كه سر و تهشون معلوم نيست؟ 
صرفا بايد بررسي كرد.

**Reply by Mohammeddsh1** [reply]
من توی این شرکت هستم و تعصبی هم ندارم
پروژه هایی که ران کرده مثل بازی و ربات تریدر یه نشونه خوب و مثبته
و هیستوری ترید زنده رو هم برای دوست من فعال کرده و من دیدم میشه لطفا توضیح بدید در مورد اینها؟؟

**Reply by mkordian** [reply]
من هم تازه تو این شرکت سرمایه‌گذاری کردم ، خیلی هم عالی هستن، نفر شاخه بالایی من داره از این شرکت و زیرشاخه هاش ماهی ۱۵۰۰ دلار درآمد درمیاره، و همینطور نفرات چند شاخه بالاتر از اون دارن ماهیانه تا۱۰۰ هزار دلار درمیارن، من تاییدشون میکنم، شرکت و دامنه وبسایتشون تو سوییس ثبت شده.

**Reply by alireza539** [reply]
خب دارم کم کم تحريک ميشم

**Comment by micro73** [comment]
در اخرین اطلاعات دریافتی پرزنتور ها با دادن صفته و چک یا زمین به مبلغی که سرمایه گذاری میکنند در حال افزایش درصد اطمینان هستن.

**Comment by davoodkiani** [comment]
سرمایه گذاری دیجیتالی نیاز به ریسک دارد
در حالی که اگر ریسک پذیری را از زندگی حذف کنیم به روالی یکنواخت و خسته
کننده می‌رسیم. در بازارهای مالی هم ریسک می‌تواند برای شما آورده مثبتی داشته
باشد. اگر انواع ریسک در سرمایه گذاری  را شناخته و با معیارهای سنجش ریسک
آشنا باشید.ومشخص است که این دسته از افراد به طور ذاتی ابایی از ریسک کردن
ندارند. کسانی که برای به دست آوردن نتیجه‌ای بهتر حاضراند کمی خطر
کنند....سرعت تبدیل دارایی به پول نقد از جمله مهم‌ترین ویژگی‌های یک
سرمایه‌گذاری خوب است. بازار سرمایه این قابلیت را دارد که در شرایط مناسب، به
سرعت دارایی شما را به پول نقد تبدیل کند



سلام به همه دوستان این مسئله رو که این شرکت پولمون رو نمیدونم میخوره و
میبره به گفته‌ یکی از دوستان که این بحث رو آغاز کرد من دو هفته پیش پاسخ
دادم  این رو به همه دوستانی  که سرمایه گذاری کردن میگم من خودم در حال حاظر
با قرارداد دو ساله و دلار 26000 تومان با سرمایه 2000 دلاری  ثبت نام کردم
الان در  حدوده 9ماه می باشد  و یک ماه دیگر کل سرمایه خودم را  گرفتم. و همه
همینطور  هسن  آخر کلام‌ واقعا جا داره اینو بگم من خودم زیر گروه هستم و
ااصلا صحبت تبلیغ و تعریف نیست  ولی  نفر اول تیم ما حدود 2سال پیش با یه
موتور پیشرو ایرانی تردد می کرد خدارو شاهد می گیرم الان ماشین زیر پاش BMW
هست به قیمت  میلیارد که این شخص وقتی سمت هر کسی میره و پیشنهاد پیوستن‌ رو
میده بدون کوچکترین  تردید طرف ثبت نام‌ میکنه..امیدوارم که هر کسی در این
سایت  سرمایه گذاری کرده به خواستش برسه و ترس از دست دادن پول رو نداشته باشه
..

**Reply by qasemi57** [reply]
دوست عزیز سلام این ماه ترینوست سودی به شما پرداخت کرده؟ من هم سرمایه گذاری کردم ولی این ماه خبری نیست واقعا نگران هستم شما خبری داری؟

**Reply by qasemi57** [reply]
منظورم اینه که برداشت داشتی و به ریال تبدیل کردی؟

**Comment by davoodkiani** [comment]
این آموزشی را که شما حرفش رو میزنی احتمالا تو سایت های شما فقط میشه یاد
گرفت .؟؟؟.

دریافت Outlook برای Android <https://aka.ms/AAb9ysg>
------------------------------
*From:* ahmad ***@***.***>
*Sent:* Saturday, September 17, 2022 11:02:13 PM
*To:* Webamoozcom/warning-list ***@***.***>
*Cc:* Davood Kiaini ***@***.***>; Comment <
***@***.***>
*Subje5ct:* Re: [Webamoozcom/warning-list] [Scam Alert] Treenvest تری ان
وست (Discussion #29)


سرمايهگذاري نياز به آموزش و تجربه كافي دارد
شرايط و قوائد سرمايه گذاري رو بايد اموزش ديد
من و شما قرارنيست براي كسي مشخص كنيم كجا بايد سرمايه گذاري كرد!
هر شخص براي سرمايه گذاري بايد حداقل قوائد رو رعايت كند
سوالي دارم،هميشه پول دست اشخاصي ميدي كه نميشناسي و نديدي؟
يا پلتفرم هايي كه سر و تهشون معلوم نيست؟
صرفا بايد بررسي كرد.

—
Reply to this email directly, view it on GitHub
<https://github.com/Webamoozcom/warning-list/discussions/29#discussioncomment-3670396>,
or unsubscribe
<https://github.com/notifications/unsubscribe-auth/AZXY6KRFVZ7I7YPHHTQXFJLV6YFC3ANCNFSM5DZZR5EQ>
.
You are receiving this because you commented.Message ID:
***@***.***>

**Reply by micro73** [reply]
سایت‌های معتبر زیاده منابع هم گسترده.
منم جایی نگفتم بیایید درسایت‌های ما یا سایت من دوره بخرید
برید بگردید منابع خارجی 
منابع ایرانی هم زیادع
منم سایت و دوره ندارم

**Reply by micro73** [reply]
ایمیل گیت‌هاب رو کلا فوروارد کردی اینجا!

**Comment by ahadmekan55** [comment]
دوست عزیز خیلی هم  عالیه  هرماه به دلار یا تتر میتونی سودت رو برداشت کنی .از جمهوری اسلامی دزد خیلی بهتره من خودم ۱۰۰ میلیون گذاشتم ماهی ۱۰ میلیون میگیرم تازه دلار هم میره بالا ۱۲میلیون هم میشه من این برج ۱۲میلیون گرفتم چون دلار رفت بالا یکی هم غریبه میخواد دست مردم بگیره شما میخوای خرابش کنی ما که از این رژیم غاصب عرب عربپرست کلاّ نا امیدیم .

**Reply by micro73** [reply]
نوش جان موفق باشید

**Reply by qasemi57** [reply]
شما این ماه سود گرفتی ازین شرکت؟

**Reply by qasemi57** [reply]
منظورم اینه که برداشت داشتین؟ به ریال تبدیل شده؟
چون منم برداشت کردم ولی در واقعیت مثل ماههای قبل به ریال تبدیل نشده و در حد مجازی مونده خیلی نگرانم چون سرمایه کمی نیست برای من

**Reply by tnztzbfm** [reply]
> شما این ماه سود گرفتی این شرکت؟

اره

**Comment by Blockunity** [comment]
تکمیل شده این گزارش در لیست هشدار @Blockunity گذاشته شده

https://github.com/Blockunity/Warning-List/discussions/4

**Reply by tnztzbfm** [reply]
کص نگو بابا من 28 فبریه برداشت کردم مشکل نداره

**Comment by ahadmekan55** [comment]
آره داداش 

Sent from Yahoo Mail on Android 
 
  On Wed, Nov 16, 2022 at 2:10, ***@***.***> wrote:   


شما این ماه سود گرفتی ازین شرکت؟

**Comment by younes143** [comment]
سلام عزیزان بنده حدود۷ماه هست ۱۰۰میلیون تومان ازطریق یکی ازدوستان که معرف بودن سرمایه گذاری کردم ماهی ۱۰تومن درآمددارم

**Comment by younes143** [comment]
والله ماهم داریم میگیریم فعلا

**Comment by younes143** [comment]
تابه الان که پرداخت کردن به موقع

**Comment by qasemi57** [comment]
سلام الان حدود 4 ماه میشه که حق برداشت نداره کسی و نمیشه برداشت کرد، شما
مگه برداشت کردید؟

**Reply by tnztzbfm** [reply]
آره

**Comment by micro73** [comment]
نقد و بررسی جدید سایت میهن بلاکچین در مورد پروژه Treenvest، خواندنی است.
اگر سرمایه‌گذاری کردید ویا تصمیم به سرمایه‌گذاری دارید این مطلب به شدت خواندنی است و پیشنهاد میشه 
برای افرادی که شاید کمی تفکر کنند.

https://mihanblockchain.com/review-of-treenvest-scam/

**Comment by tnztzbfm** [comment]
کی گفته کلاورداری من که راضیم

**Reply by tnztzbfm** [reply]
من پری روز برداشت کردم مشکلی ندارد سایت تیرینوست

**Reply by tnztzbfm** [reply]
منم

**Comment by tnztzbfm** [comment]
من که از سایت تیرینوست پری روز برداشت کردم مشکلی ندارد 10میلیون گرفتم ایرانی دوزده نه سایت تنرینوست

**Comment by qasemi57** [comment]
خوش به حالتون ما که ۵ ماهه حدودا نمیتونیم برداشت کنیم نمیدونم چرا ولی به
مشکل خوردیم شما اطلاعی ندارین این مشکلی که در ترینوست برای بعضی سرمایه
گذارا پیش اومده حل شدنی هست یا نه؟!! تورو خدا اگر کسی چیزی میدونه اطلاع
رسانی کنه

**Reply by tnztzbfm** [reply]
مشکلی ندارن👍🏦

**Reply by tnztzbfm** [reply]
> خوش به حالتون ما 5 ماه حدود نمیتونیم نمیتونیم بفهمیم چرا ولی به مشکل بخوریم شما اطلاعی ندارین این مشکلی که در بیشتر از قبل سرمایه دار شدن پیش حلی هست یا نه؟!! تورو خدا اگر کسی چیزی میدونه اطلاع رسانی کنه
> […](#)
> On Fri, Mar 3, 2023, 19:34 tnztzbfm ***@***.***> wrote: من که از سایت تیرینوست پری روز برداشت کردم مشکلی ندارد 10میلیون گرفتم ایرانی دوزده نه سایت تنرینوست — Reply to this email directly, view it on GitHub <[#29 (comment)](https://github.com/Webamoozcom/warning-list/discussions/29#discussioncomment-5194555)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A4GUWRTURMKXXPHU34LG37DW2II7FANCNFSM5DZZR5EQ> . You are receiving this because you commented.Message ID: ***@***.***>

میدونی چرا شما دلیل لینک  مجموعه گیری
نباید هرجا بزاری بخاطر این هست

**Reply by tnztzbfm** [reply]
حساب شما به مدت یک سال مسدود شده بعد از یک سال باز میشه

**Comment by tnztzbfm** [comment]
> **نام پروژه:** Treenvest
> 
> **نام پروژه:** تری آن وست
> 
> **پروژه نماد:**
> 
> **اتفاقی که افتاده است:** شرکت treenvest نیز در رسته شرکت های پانزی قرار می گیرد و همچنین می تواند از پلن های درامدآوری در رسته شرکت های هرمی قرار گیرد با گستردگی فعالیت ها در کامیونیتی های فارسی زبان می تواند این شرکت را در لیست قرار دادن ماهیان ثابت کند. می دهند.
> 
> **قوانینی که تصویب شده است:**
> 
> **نوع هشدار:** هشدار کلاهبرداری
> 
> **نوع کلاهبرداری:** طرح پانزی
> 
> **وضعیت کلاهبرداری:** در جریان
> 
> **نام کلاهبرداران:**
> 
> **نام شرکت(ها):**
> 
> **آدرس وب سایت(ها): ** [https://www.treenvest.com/ ](https://www.treenvest.com/) [https://t.me/Treenvestir ](https://t.me/Treenvestir) [https://t.me/Treenvestiran ](https://t.me/Treenvestiran) [https://www.instagram.com/iran.treenvest/? hl=fa ](https://www.instagram.com/iran.treenvest/?hl=en) [https://instagram.com/iran.treenvest?utm_medium=copy_link ](https://instagram.com/iran.treenvest?utm_medium=copy_link) https://instagram.com/treenvestir?utm_medium=copy_link
> 
> **مبلغ کلاهبرداری:**
> 
> **منابع:**
> 
> **جمع بندی:**

ببین کص مادر چرا دروغ میگی من تا الان پول گرفتم هیچ مشکلی ندارن کلاهبردار باباته

**Comment by tnztzbfm** [comment]
بچه‌ها سایت هیچ مشکلی نداره خیالت راحت باشه سرمایه گذاری کنید 💪👍🏦

**Reply by micro73** [reply]
صحیح...
حتما دلایل مشکل نداشتن و خوب بودن رو عنوان کنید که متوجه بشیم.
دلایل مشکل داشتن رو ما گفتیم.

**Comment by micro73** [comment]
در بررسی‌های صورت گرفته جدید مشخص شده که به جای سوددهی دلاری به کاربران از ارز 3co (مشکلات و سطرهای کلاهبرداری قراردادهوشمند این توکن در پست‌های قبلی بررسی شده) استفاده می‌کنند. 

همچنین تعدادی از افراد مجموعه‌های مختلف برای ضمانت خوب بودن این شبکه از چک ضمانت استفاده میکنند که بعدا گریبان گیر خودشان میشود.

**Reply by tnztzbfm** [reply]
> در بررسی‌های صورت گرفته جدید مشخص شده است که به جای سوددهی دلاری به کاربران از ارز 3co (مشکلات و سطرهای کلاهبرداری قراردادهوشمند این توکن در پست‌های قبلی بررسی شده) استفاده می‌کنند.
> 
> همچنین از افراد مجموعه‌های مختلف برای ضمانت خوب بودن این شبکه از چک ضمانت استفاده می‌کنند که بعدا گریبان گیر می‌شود.

باز کص گفتی تو؟؟

**Reply by tnztzbfm** [reply]
مگه کونت میخایه؟؟

**Comment by tnztzbfm** [comment]
توضیحات کامل در کانال تلگرم
https://t.me/TreenvestIra

**Reply by micro73** [reply]
بعد از سه روز منتظر پاسخ مدیر این کانال به سوالاتم هستم.
ایدی مدیر رو هم برداشتید از کانال که...
اگه مدیرهاتون نمیتونن، شما بیا پاسخ بده به سوالات. باید اطلاعاتت خیلی بالا باشه...

ما سندو مدرک داریم 
شماهم با سند و مدرک بیا...

**Reply by micro73** [reply]
خب مدیر همین کانالی که معرفی کردی پاسخ داد... اما با فحاشی

امیدوارم به اگاهی کامل برسید.
به سوالات که بلد نبود جواب بده و چیزی جز فحاشی یادش ندادن، چیزی که خودتم باید یادبگیری سالهای سال...

هزارتا کانال و گروهم اینجا بزاری بازم مشخصه پروژه کلاهبرداریه...

اگه میتونی خلافشو با سند ثابت کن، ما سند داریم کلاهبرداری است.

@Jll_il_s مدیر کانال شماره یک که فحاشی کرد و بلاک کردن - همچنین ایدی ایشون از کانال برداشته شد.
https://t.me/treenvest3c0

 @Jll_il_a در انتظار پاسخگویی مدیر شماره ۲ هستم 
https://t.me/treenvest6

👋 مشخص است که هردو کانال برای اعضای یک تیم بوده که توانایی پاسخگویی به کوچیکترین سوالات رو ندارند ولی در حال فریب مردم هستند... 

مدیر کانال شماره ۲ هم بدون پاسخگویی بلاک کردند.

مراقب چنین پروژه‌های بی‌هویت باشید که هیچ کس توان پاسخ گویی ندارد.

**Comment by tnztzbfm** [comment]
توضیحات ترینوست https://t.me/TreenvestIra

**Comment by tnztzbfm** [comment]
توضیحات کامل در کانال تلگرامhttps://t.me/TreenvestIra
